src/routers/comment.js

```diff
@@ -0,0 +1,17 @@
+const {Router} = require('express');
+const router = Router();
+const checkAuth = require('../middleware/auth');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const checkAuth = require("../middleware/auth");
```